src/triage/component/catwalk/model_testers.py

```diff
@@ -19,6 +18,7 @@ def __init__(
         individual_importance_config,
     ):
         self.matrix_storage_engine = matrix_storage_engine
+        self.db_engine = db_engine
```

Comment:
You might have another reason for setting this on `ModelTester`, but it doesn't appear to be (newly) in use.

Reply:
Nope, engindectomy performed